test/common/upstream/health_checker_impl_test.cc

```diff
@@ -3564,6 +3702,56 @@ class GrpcHealthCheckerImplTestBase : public GrpcHealthCheckerImplTestBaseUtils
     addCompletionCallback();
   }
 
+  void expectSessionCreate() {
+    // Expectations are in LIFO order.
+    TestSessionPtr new_test_session(new TestSession());
+    test_sessions_.emplace_back(std::move(new_test_session));
+    TestSession& test_session = *test_sessions_.back();
+    test_session.timeout_timer_ = new Event::MockTimer(&dispatcher_);
+    test_session.interval_timer_ = new Event::MockTimer(&dispatcher_);
```

Comment:
You can move these after the definition of `new_test_session` and create the timers for that variable (instead of the definition of `test_session` that references the same object).
This can also be done in `HttpHealthCheckerImplTest`

Reply:
Done.